plugins/uberdev/lib/code_fixer_contract.py

```
@@ -309,6 +314,350 @@ def _enveloped_body(payload: bytes, source: str) -> bytes:
+    return -sum(
+        (count / total) * math.log2(count / total) for count in Counter(run).values()
+    )
+
+
+def _class_churn(run: str) -> float:
+    """Share of `run`'s adjacent character pairs that change class.
+
+    `run` must be at least two characters. Every caller passes a run already
+    held to `_CONVENTION_SECRET_CORE_MIN`, so the pair list is never empty.
+    """
+    pairs = list(zip(run, run[1:]))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)